docs/data/toolpad/getting-started/quickstart.md

```diff
@@ -0,0 +1,129 @@
+# Quickstart
+
+<p class="description">Get started building a Toolpad app.</p>
+
+This page will guide you into creating your very first Toolpad application. You will create a "hello world" type of application that will ask for a name and print a greeting.
+
+## Prerequisites
+
+Make sure you have a running Toolpad instance.
+
+1. Download the docker compose file
+
+   ```sh
+   curl -LO https://raw.githubusercontent.com/mui/mui-toolpad/master/docker/compose/docker-compose.yml
+   ```
+
+1. Start the docker compose services
+
+   ```sh
+   docker-compose -f docker/compose/docker-compose.yml up -d
```

Comment:
Based after https://github.com/mui/mui-toolpad/pull/859/files#r968773969, I could reproduce.

```suggestion
   docker-compose -f docker-compose.yml up -d
```

Reply:
This is **off-topic** for this PR but it could be interesting for the onboarding experience:

1. I wonder if it would make sense to ask to create an empty folder.
2. The amd64 emulator is super slow on a MacBook M1/M2

<img width="315" alt="Screenshot 2022-09-12 at 21 48 10" src="https://user-images.githubusercontent.com/3165635/189742558-e1cabd23-0e5d-44c8-ba36-7e8a36d1f348.png">

It's not usable, e.g.

<img width="743" alt="Screenshot 2022-09-12 at 21 49 39" src="https://user-images.githubusercontent.com/3165635/189742830-e2349790-d05a-4134-90c7-040f29b990cd.png">

3. I don't have the drop component here

<img width="897" alt="Screenshot 2022-09-12 at 21 49 58" src="https://user-images.githubusercontent.com/3165635/189742892-4672a262-29f1-4f26-ab19-589f2b63888d.png">

Reply:
Created this issue so we can keep track of it

Reply:
We could pick this back up at some point: https://github.com/mui/mui-toolpad/pull/618

I'm wondering if it's the real problem though. I'm on M1 and don't see the same behavior. appsmith and Toolpad hum along on Docker just fine for me.